homeassistant/components/flo/services.yaml

```diff
@@ -15,9 +15,9 @@ set_sleep_mode:
       selector:
         select:
           options:
-            - "120"
-            - "1440"
-            - "4320"
+            - 120
+            - 1440
+            - 4320
```

Comment:
Selector options are strings.

This should be handled differently in the service handler